### ISSUE TITLE
default-index.d.ts is outdated

### ISSUE THREAD

**Issue by janpio** [prisma/prisma#6500]
https://github.com/prisma/prisma/blob/master/src/packages/client/scripts/default-index.d.ts is outdated in comparison to the fully generated version:

- Code example in docs comment
- Documentation link in docs comment
- Possibly missing namespace for property dmmf?
- Line breaks for `PromiseType` and `PromiseReturnType`

Found via comparing `node_modules\.prisma\client\index.d.ts` in a project before and after generating a Client the first time.

**Comment by jasonsilvers** [comment]
I think this is related to an issue I am seeing as this does not work. 

```
import { PromiseReturnType } from '@prisma/client'
```

Maybe the [docs](https://www.prisma.io/docs/concepts/components/prisma-client/advanced-type-safety/operating-against-partial-structures-of-model-types#solution-1) are incorrect?

**Comment by pantharshit00** [comment]
@jasonsilvers Sounds unrelated. Can you please open a new issue with the usage which you are doing?

**Comment by janpio** [comment]
(@jasonsilvers Have you tried to import `Prisma.PromiseReturnType`? We indeed changed something there recently and this could indeed be outdated in the docs if that applies there as well.)

**Comment by janpio** [comment]
Please open a new issue then 🙇

**Comment by kiily** [comment]
This works:
1) Import `Prisma` namespace `import { Prisma } from "@prisma/client";` - the namespace itself seems to be a named export
2) Use as shown in the docs but from the namespace `Prisma.PromiseReturnType`

Seems to be working nicely for me 🙏

**Comment by pantharshit00** [comment]
@janpio Is this still relevant?

**Comment by janpio** [comment]
Yes, someone from the Client TS team needs to take a look at the generated vs. the stub version and confirm they are identical/equivalent.
